types/mongoose/index.d.ts

```diff
@@ -3322,6 +3329,7 @@ declare module "mongoose" {
     where(path: string, val?: any): Query<any> & QueryHelpers;
   }
 
+  class Document {}
```

Comment:
Can you add a test for instantiating `Document?

Reply:
added